apiserver/facades/client/subnets/subnets_test.go

```diff
@@ -803,6 +489,41 @@ func (s *SubnetsSuite) TestAddSubnetsWhenNetworkingEnvironNotSupported(c *gc.C)
 	)
 }
 
+func (s *SubnetsSuite) TestAddSubnetAPI(c *gc.C) {
```

Comment:
Added to have 1 successful AddSubnet test from the subnet facade.  Handling the params combinations is now solely tested in networkingcommon_test.